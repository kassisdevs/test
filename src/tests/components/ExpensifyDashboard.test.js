import React from 'react';
import { shallow } from 'enzyme';
import ExpensifyDashboard from '../../components/ExpensifyDashboard';

test('Sould render dashboard two components', () => {
  const wrapper = shallow(<ExpensifyDashboard />);
  expect(wrapper).toMatchSnapshot();
});