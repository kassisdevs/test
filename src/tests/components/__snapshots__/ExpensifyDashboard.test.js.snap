// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sould render dashboard two components 1`] = `
<div>
  <Connect(ExpenseSammary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
